B9PartSwitch/PartSwitch/ModuleB9PartInfo.cs

```diff
@@ -49,7 +49,12 @@ private void Start()
             SetupGUI();
             UpdateFields();
 
-            GameEvents.onEditorShipModified.Add(EditorShipModified);
+            GameEvents.onEditorShipModified.Add(this.EditorShipModified);
```

Comment:
Isn't `this` implicit here since it's an instance method?

Reply:
Yes, it's an instance method.

Using explicit `this` is an old habit of mine, the compiler would solve the thing by itself - but using `this` is a very visual way to tell the difference between a instance method and a static method.

Some event handlers takes a static pointer, and others wants an instance method, and visually differentiating them is easier than trying to remember the documentation.

In a way or another, it's a habit of mine, not exactly a "best practice" no there's no real need of them.

I'm amending the pull request removing them.